delta-api/src/main/java/io/cdap/delta/api/Offset.java

```diff
@@ -53,7 +53,7 @@ public boolean equals(Object o) {
   @Override
   public String toString() {
     if (state.isEmpty()) {
-      return null;
+      return "";
     }
```

Comment:
Let's remove this, it's fine to just call state.toString() even if state is an empty map